internal/platform/merkletree/merkletree.go

```diff
@@ -0,0 +1,20 @@
+// Package merkle provides implementations for merkle tree.
+package merkletree
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// Service defines the interface for storage initialization and termination.
```

Comment:
This comment refers to another struct, change it please

Reply:
changed